flycheck.el

```diff
@@ -8788,6 +8789,26 @@ See URL `https://docs.python.org/3.4/library/py_compile.html'."
           line ", " column ", " (one-or-more not-newline) line-end))
   :modes python-mode)
 
+(flycheck-def-args-var flycheck-python-mypy-args python-mypy)
+
+(flycheck-define-checker python-mypy
+  "Mypy syntax and type checker.  Requires mypy>=0.3.1.
+
+Customize `flycheck-python-mypy-args` to add specific args to default
+executable.
+
+E.g. when processing Python 2 files, add \"--py2\".
+
+See URL `http://mypy-lang.org/'."
+
+  :command ("mypy"
+            (eval flycheck-python-mypy-args)
+            source-original)
```

Comment:
mypy supports stdin, so we should use that.

Reply:
How do you feel about using `source-inplace` here instead? That way mypy can follow imports too.

Reply:
mypy can follow imports if you give it --shadow-file, and this way we don't need to pollute the current folder with `flycheck_`-prefixed files